front/components/Layout/Header/SearchBar/index.tsx

```diff
@@ -18,9 +18,21 @@ function SearchBar({ className }: SearchBarProps): JSX.Element {
   }
 
   return (
-    <Formik onSubmit={handleSubmit} initialValues={{ search }}>
-      <Form className={className}>
-        <TextInput disableHelperText fullWidth name="search" type="search" placeholder="search..." />
+    <Formik
+      onSubmit={handleSubmit}
+      initialValues={{ search }}
+    >
+      <Form
+        className={className}
+        data-testid="SearchBarInput"
```

Comment:
I think we should not need to have a specific `data-testid` here and that a `getByRole` should be sufficient.